app/javascript/dashboard/routes/dashboard/contacts/components/ContactInfoPanel.vue

```diff
@@ -3,7 +3,7 @@
     <span class="close-button" @click="onClose">
       <i class="ion-android-close close-icon" />
     </span>
-    <contact-info show-new-message :contact="contact" />
+    <contact-info show-new-message :contact="contact" @onPanelClose="onClose" />
```

Comment:
Can we use camel case for @onPanelClose ? Or simply rename to => @close or @panel-close

Reply:
Renamed to `@panel-close`